de.json

```diff
@@ -1906,42 +1906,42 @@
-  "speed_camera_type_bus_lane": "Bus lane camera",
-  "speed_camera_type_dangerous_zone": "Speed camera zone",
-  "speed_camera_type_mobile": "Mobile speed camera",
-  "speed_camera_type_red_light": "Red light camera",
-  "speed_camera_type_red_light_and_speed": "Speed and red light camera",
-  "speed_camera_type_regular_mobile_zone": "Mobile speed control zone",
-  "speed_camera_cm_zone_ending_short": "Control zone ending",
-  "speed_camera_cm_zone_starting_short": "Average speed control",
-  "log_in_with_qr_step_1": "Launch the ABRP app on a device where you are already logged in",
-  "log_in_with_qr_step_2": "Go to the settings page",
-  "log_in_with_qr_step_3": "Select your name in the top left",
-  "feedback_charger_image": "Can you take a photo of the charger?",
-  "continue_on_phone": "Continue on your phone",
-  "general_connection_error_title": "Connection error",
-  "general_connection_error_description": "Please try again and if the issue persists, please contact support.",
-  "store_review_title": "Enjoying ABRP?",
-  "store_review_description": "We would like to know if you like the app. Your feedback will help us to make a difference!",
-  "love_it": "Love it",
-  "needs_improvement": "Needs improvement",
-  "rate_us_on_app_store": "Rate us on the App Store",
-  "rate_us_on_play_store": "Rate us on the Google Play Store",
-  "help_us_improve": "Help us improve",
-  "feedback_improve_description": "Your insights help us grow. Tell us what didn't meet your expectations.",
-  "please_share_your_thoughts": "Please share your thoughts...",
-  "store_review_not_available": "Store review is not available on this device."
-}
\ No newline at end of file
+  "upload_photos": "Fotos hochladen",
+  "take_new_photos": "Neue Fotos aufnehmen",
+  "add_charger_image_detailed_description": "Versuche, das Ladegerät und die Umgebung zu fotografieren. So können andere das Ladegerät finden und wissen, was sie erwartet.",
+  "add_charger_image_anonymization": "Wir werden Kennzeichen und Gesichter automatisch unkenntlich machen.",
+  "speed_camera_category": "Blitzer",
+  "speed_camera_premium_title": "Eine Warnung vor einem bevorstehenden Blitzer anzeigen",
+  "speed_camera_premium_text": "Blitzer auf der Karte anzeigen",
+  "speed_camera_on_screen_alert": "Warnungen auf dem Bildschirm",
+  "speed_camera_on_screen_alert_description": "Anzeige von Warnhinweisen auf dem Bildschirm vor einem Blitzer",
+  "speed_camera_map_markers": "Kartenmarkierungen",
+  "speed_camera_map_markers_description": "Markierungen für Blitzer auf deiner geplanten Route anzeigen",
+  "speed_camera_type_speed": "Geschwindigkeitsblitzer",
+  "speed_camera_type_average": "Überwachung der Durchschnittsgeschwindigkeit",
+  "speed_camera_type_bus_lane": "Überwachung der Busspur",
+  "speed_camera_type_dangerous_zone": "Blitzer-Zone",
+  "speed_camera_type_mobile": "Mobiler Blitzer",
+  "speed_camera_type_red_light": "Rotlichtblitzer",
```

Comment:
Das ist ein Ampel Blitzer. Rotlichblitzer hört sich irgendwie seltsam an...

Reply:
Hab mal eine schnelle Analyse gemacht: [OSM](https://wiki.openstreetmap.org/wiki/DE:Proposed_features/Relation:enforcement#Beispiel_2:_Rotlicht-Kamera_nur_f%C3%BCr_Linksabbieger) nennt es "Rotlicht-Kamera" und [Waze ](https://www.waze.com/wiki/Germany/Blitzer_einpflegen) nennt es "Rotlichtblitzer". Ich bin aber voll deiner Meinung, dass sich Rotlichtblitzer seltsam anhört. Ich bin mir aber nicht sicher ob die User in der Situation verstehen was ein "Ampel Blitzer" tut.
Hier was ich sonst [noch gefunden habe ](https://de.wikipedia.org/wiki/Rotlicht%C3%BCberwachung)(keines davon gefällt mir sehr):
Rotlichtüberwachung
Blitzampeln
Rotblitzer
Ampelradar
Rotlicht-Radar

Reply:
In der Aufzählung fählt noch:
Starenkasten ;-)

Radar ist bei Thema Ampel nicht die richtige Bezeichnung, denn es wird ja keine Geschwindigkeit gemessen sondern beim Überfahren der roten Ampel wird dieses festgestellt und geblitzt, eine Geschwindigkeit die per Radar messbar ist ist hier irrelevant.

Ich hingegen finde Ampel-Blitzer seltsam und würde Rotlichtblitzer bevorzugen.
Denn, wenn wir uns die Frage stellen, was wird eigentlich überwacht, ist die Schlußfolgerung logisch:
Die Ampel wird wird nicht generell überwacht, also sollte es nicht Ampel-Blitzer heißen.
Es wird die Ampel nur bei Rotlicht überwacht, wenn also jemand in der Rotphase die Ampel überquert. Also sollte es Rotlicht-Blitzer oder Rotlichtblitzer heißen.

Was meint ihr?

Reply:
Ich bin deiner Meinung. Auch technisch sehe ich hier Induktionsschleifen, LiDAR und Kamerasysteme, und keine Radarsysteme. Was mich wohl am Begriff gestört hat, war die Zweideutigkeit des Begriffes "Rotlicht". Bin aber aus Usability (schnelle Erfassung durch den User was er tun/lassen soll) und technischer Sicht der Meinung, dass Rotlichtblitzer passt.

Reply:
Also dann einigen wir uns auf Begriff Rotlichtblitzer?

Reply:
Ok